includes/User.php

```diff
@@ -36,7 +36,7 @@
  * Int Serialized record version.
  * @ingroup Constants
  */
-define( 'MW_USER_VERSION', 10 );
+define( 'MW_USER_VERSION', 11 );
```

Comment:
We've done this a few times with the preferences ramp (the value when we started was `9`), so loading the data from the DB isn't really a problem. I'm bumping the version here to force a sync between what's in the DB and what's in memcache, so that any issues with this change are immediately visible instead of delayed to when the memcache key expires in the cache.